### ISSUE TITLE
clipboard sharing not working under virtualbox 6.1.4

### ISSUE THREAD

**Issue by ltalirz** [marvel-nccr/quantum-mobile#112]
for some reason, clipboard sharing no longer works under virtualbox 6.1.4

others also seem to have experienced this issue:
https://forums.virtualbox.org/viewtopic.php?f=2&t=96196

**Comment by ltalirz** [comment]
As mentioned in the discussion thread, a workaround is to downgrade the Virtualbox GuestAdditions in the guest system to 6.1.2 (I have confirmed this).

Since the current image we're using already ships with VirtualBox GuestAdditions 6.1.2, a simple solution for the moment is to build the VM with
```
vagrant plugin uninstall vagrant-vbguest
```

Mentioning @giovannipizzi  and @qiaojunfeng for info

**Comment by giovannipizzi** [comment]
Here is the ticket https://www.virtualbox.org/ticket/19336
Apparently there is already a downloadable fix, I imagine there will be a new version released soon.

We might want to wait a few days to see if it gets released?

**Comment by ltalirz** [comment]
Right - let's wait a bit. If it isn't released, we can install it manually

**Comment by ltalirz** [comment]
So, since there doesn't seem to be an official new VBox release for the moment, here the current workaround:

```
wget https://www.virtualbox.org/download/testcase/VBoxGuestAdditions_6.1.97-136310.iso
sudo mount VBoxGuestAdditions_6.1.97-136310.iso /mnt
sudo /mnt/VBoxLinuxAdditions.run
sudo vim /etc/systemd/system/vboxadd-service.service # replace 6.1.4 by 6.1.97
sudo reboot
```

Works with VirtualBox 6.1.4 on the host.

**Comment by chrisjsewell** [comment]
This was fixed in 6.1.6
